src/test/java/com/wikia/webdriver/common/logging/PageObjectLogging.java

```diff
@@ -191,12 +209,12 @@ public static void logImage(String command, File image, boolean success) {
     }
     logImage(command, new String(bytes, StandardCharsets.UTF_8), success);
   }
-
+  //TODO: Check error class here
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1135)